projectroles/templates/projectroles/project_form.html

```diff
@@ -57,7 +57,7 @@ <h2>Create {{ project_display }} or {{ category_display }} Under {{ parent.title
     {% elif disable_categories %}
       <h2>Create {{ project_display }}</h2>
     {% else %}
-      <h2>Create Top Level {{ category_display }}</h2>
+      <h2 id="object-type">Create Top Level {{ category_display }}</h2>
```

Comment:
Comparing against freely entered strings is always a bit risky. I'd recommend instead to set e.g. a `data-project-action` attribute here and set some short descriptor there. That way, we avoid the risk of the JQuery breaking if we decide to reword the displayed title for the view.